lib/asciidoctor/converter/html5.rb

```diff
@@ -1,3 +1,5 @@
+require 'yaml'
```

Comment:
I don't think it's a good idea to use the `yaml` module because I doubt it will work in Asciidoctor.js.

Reply:
Okay, I try to another plan.

I intend to define `svg` .And write `svg` .